osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerPositionDisplay.cs

```diff
@@ -158,13 +158,15 @@ private void updatePosition()
                 return;
             }
 
-            float relativePosition = (float)(position.Value.Value - 1) / scores.Count;
+            float relativePosition = Math.Clamp((float)(position.Value.Value - 1) / (scores.Count - 1), 0, 1);
```

Comment:
This falls over when there's only one player because of NaNs that `Math.Clamp()` will just propagate forward. Probably want something like

```suggestion
            float relativePosition = Math.Clamp((float)(position.Value.Value - 1) / Math.Max(scores.Count - 1, 1), 0, 1);
```

here. And a test maybe, not sure, your call.